walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,174 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
 public final class FindMeetingQuery {
+  private List<TimeRange> attendeesEventTimes;
+  private Map<String, List<TimeRange>> optionalAttendeesEventTimes;
+  private List<List<String>> optionalAttendeesCombs;
+
+  /** 
+   * Finds a list of times when the requested event can happen, 
+   * given the list of current events and the request information. 
```

Comment:
Can you add some more details about the conditions?

Also line wrapping.

Reply:
Sure. What is wrong with the line wrapping?